packages/openapi-to-graphql/src/utils.ts

```diff
@@ -13,9 +13,6 @@ export const mitigations = {
    * Should be caught by the module oas-validator
    */
   INVALID_OAS: `Ignore issue and continue.`,
-  INVALID_SCHEMA_TYPE: `Fall back to GraphQL string type and stringify returned data.`,
```

Comment:
These errors are not used, are redundant, and/or are handled by other mitigation methods (like #186).